docs/api/geo_datasets.csv

```diff
@@ -0,0 +1,17 @@
+Dataset,Type,Source,Size (px),Resolution (m)
+`Aboveground Woody Biomass`_,Mask,"Landsat, LiDAR","~40,000x40,000",~30
```

Comment:
Would it look better with "40,000 x 40,000" with a space around the x? I wish we could do `$\times$` but I don't think that works here.

Reply:
Let's see!

Reply:
Old: ![image](https://user-images.githubusercontent.com/1088078/174436102-ab577a48-fcc5-40f2-8016-868e3e8d3335.png)

Reply:
Not seeing any change when I add spaces actually.

Reply:
Using &nbsp; didn't work either